romana-install/roles/romana/install-master/templates/config/romana.conf.yml

```diff
@@ -84,6 +84,6 @@ services:
     config:
       kubernetes_url : "http://{{ romana_master_ip }}:8080"
       namespace_notification_path: "/api/v1/namespaces/?watch=true"
-      policy_notification_path_prefix : "/apis/romana.io/demo/v1/namespaces/"
-      policy_notification_path_postfix : "/networkpolicys/?watch=true"
-      segment_label_name: "segment"
+      policy_notification_path_prefix : "/apis/extensions/v1beta1/namespaces/"
+      policy_notification_path_postfix : "/networkpolicies/?watch=true"
```

Comment:
Yes. Kubernetes has a funny way of converting third-party resource names into RESTful paths.
Eg: it takes `name.domain.com` and turns it into `/apis/domain.com/.../names` (adding `s` to it). For some words, that didn't work very well. Now it's a first-class object, it can be properly named without that automatic conversion.